stdlib/filepath/src/main/ballerina/src/filepath/file_path.bal

```diff
@@ -25,9 +25,12 @@ string pathSeparator = isWindows ? "\\" : "/";
 string pathListSeparator = isWindows ? ";" : ":";
 
 # Retrieves the absolute path from the provided location.
+# ```ballerina
+#  string absolutePath = checkpanic filepath:absolute(<@untainted> "test.txt");
+# ```
 #
-# + path - String value of file path.
-# + return - The absolute path reference or an error if the path cannot be derived
+# + path - String value of the file path
```

Comment:
I think it's better if we can explain the reasoning behind why this param is marked as `@untainted`.

Reply:
Corrected.